spacetemplate/assets/agile.yaml

```diff
@@ -89,36 +89,20 @@ work_item_types:
         - In Progress
         - Resolved
         - Closed
-    "effort":
-      label: Effort
-      description: 
-        The effort needed to implement this theme.
+    "business_value":
+      label: Component
```

Comment:
@michaelkleinhenz @stevengutz Why label the field `business_value` with `Component`?

Reply:
The label has a cut-and-paste error. Fixed.